conda-forge.yml

```diff
@@ -1,4 +1,4 @@
 conda_forge_output_validation: true
 provider: {linux_aarch64: false, linux_ppc64le: default}
 choco:
-    - cuda --version=10.2.89 --skip-powershell
+    - cuda --version=11.0.3
```

Comment:
Did we want `10.1` here?

Reply:
I have tried that and choco failed to install

Reply:
Ah ok. Interesting. Thanks for the info :)